backend/matching-service/spec/matching-queue.spec.js

```diff
@@ -0,0 +1,75 @@
+const MatchingQueue = require('../models/matching-queue');
+const Match = require('../models/match.object');
+
+describe('MatchingQueue', () => {
+  let queue;
+
+  beforeEach(() => {
+    queue = new MatchingQueue();
+  });
+
+  describe('#enqueue', () => {
```

Comment:
Should we also standardise between:

`describe('#{function_name}')`

and

`describe('{function_name} function')`

?

Reply:
Changed in the most recent commit!